src/Migrations/M028MovePageHeaderSideBarOptions.php

```diff
@@ -0,0 +1,222 @@
+                    'p4_hide_page_title_checkbox',
+                );
+
+                // Update specific meta values to empty strings
+                foreach ($keys_to_update as $meta_key) {
+                    if ($meta_key === 'p4_hide_page_title_checkbox' && !empty($page_header_title)) {
+                        if (isset($page_meta_data[$meta_key]) && in_array('on', $page_meta_data[$meta_key])) {
+                            continue;
+                        }
+
+                        // Update meta value with string 'on'
+                        update_post_meta($page->ID, $meta_key, 'on');
+                    } else {
+                        update_post_meta($page->ID, $meta_key, '');
+                    }
+                }
+
+
+                $post_args = array(
+                    'post_content' => $updated_post_content,
+                    'ID' => $page->ID,
+                );
+
+                wp_update_post($post_args);
+
+                continue;
+            }
+
+            if (empty($page_header_image_id) || empty($page_header_title)) {
+                if (
+                    empty($page_header_image_id)
+                    && empty($page_header_title)
+                    && empty($page_header_description)
+                    && empty($page_header_button_title)
+                ) {
+                    continue;
+                }
+
+                $content = '
+                    <!-- wp:group {"layout":{"type":"flex","orientation":"vertical"}} -->
+                        ' . (!empty($page_header_title) ?
+                            '<div class="wp-block-group"><!-- wp:heading -->
+                                <h1 class="wp-block-heading">' . $page_header_title . '</h1>
+                            <!-- /wp:heading -->'
+                        :
+                        '') . '
+
+                        ' . (!empty($page_header_description) ?
+                            '<!-- wp:paragraph -->
+                                <p>' . $page_header_description . '</p>
+                            <!-- /wp:paragraph -->'
+                        :
+                        '') . '
+
+                        ' . (!empty($page_header_button_link) ?
+                            '<!-- wp:buttons -->
+                                <div class="wp-block-buttons"><!-- wp:button -->
+                                <div class="wp-block-button is-style-cta">
+                                    <a class="wp-block-button__link wp-element-button"
+                                        href="' . $page_header_button_link . '"
+                                        target=' . ($page_header_button_link_checkbox ? "_blank" : "") . '
+                                        rel="noreferrer noopener"
+                                    >
+                                        ' . $page_header_button_title . '
+                                    </a>
+                            </div>
```

Comment:
I think that `</div>` should be moved a few lines lower, next to `<!-- /wp:group -->`. This is probably why the footer looks wrong in the Terms page.